onnxruntime/core/optimizer/matmul_add_fusion.cc

```diff
@@ -13,10 +13,14 @@ namespace onnxruntime {
 Status MatMulAddFusion::ApplyImpl(Graph& graph, bool& modified, int graph_level) const {
```

Comment:
Can this become a rewrite rule with latest changes?
We can do it in another PR.

Reply:
I think so. Not much needs to be a transformer, and anything that's doing a 'simple' fusion between some nodes should be able to be a rewrite rule.

Reply:
Yeah let's do it (in another PR).